src/config/config.c

```diff
@@ -728,3 +728,16 @@ __wt_config_subgets(
 
     return (__wt_config_subgetraw(session, cfg, &key_item, value));
 }
+
+/*
+ * __wt_config_subget_next --
+ *     Get the processed value for a given key from a config string.
```

Comment:
Please extend the comment to add more context. It's for the unusual case of walking through a list, so it's not a regular key/value pair like is being retrieved by most of the configuration parsing routines.